lib/kramdown_rpf/rpf.rb

```diff
@@ -101,7 +110,7 @@ def self.convert_hints_to_html(hints)
       end
 
       def self.convert_new_page_to_html
-        ::Kramdown::Document.new("<div class=\"c-print-page-break\" />", KRAMDOWN_OPTIONS).to_html
+        ::Kramdown::Document.new('<div class="c-print-page-break" />', KRAMDOWN_OPTIONS).to_html
```

Comment:
this is changed here (disagree using single quotes for output regardless of escaping)...